index.js

```diff
@@ -1,16 +1,19 @@
-function absoluteUrl(req, setLocalhost) {
-  var protocol = 'https:';
-  var host = req ? (req.headers['x-forwarded-host'] || req.headers['host']) : window.location.host;
-  if (host.indexOf('localhost') > -1) {
-    if (setLocalhost) host = setLocalhost;
-    protocol = 'http:';
+'use strict'
+exports.__esModule = true
+function absoluteUrl(req, localhostAdress) {
+  if (localhostAdress === void 0) {
```

Comment:
TIL about `void 0`, this is just another way to say `localhostAddress` is undefined right? Is it more typescript idiomatic to compare against void 0?

```suggestion
  if (localhostAddress === void 0) {
```

Reply:
Lol, I haven't got attention to this earlier. Yea, look like it's analogue of `typeof localhostAdress === 'undefined'`. Just learned it today too lol. Turns out it's safer in some cases, here's explanation of it:

https://stackoverflow.com/a/19369074/3720305